snapshots/output/diagnostics/index.ts

```diff
@@ -0,0 +1,26 @@
+// < definition diagnostics 0.0.1 `index.ts`/
+
+/** @deprecated This class is deprecated */
+class Foo {}
+//    ^^^ definition diagnostics 0.0.1 `index.ts`/Foo#
+//    diagnostic Information:
+//    > This class is deprecated
+
+/** @deprecated This function is deprecated */
+function bar() {}
+//       ^^^ definition diagnostics 0.0.1 `index.ts`/bar().
+//       diagnostic Information:
+//       > This function is deprecated
+
+function main() {
+//       ^^^^ definition diagnostics 0.0.1 `index.ts`/main().
+  new Foo()
+//    ^^^ reference diagnostics 0.0.1 `index.ts`/Foo#
+//    diagnostic Information:
+//    > This class is deprecated
```

Comment:
It'd be good to add some test cases for multiline messages with `@deprecated` to capture whether they work/don't work, and make sure that the final formatting is correct.